setup.py

```diff
@@ -25,9 +25,9 @@
     include_package_data=True,
     install_requires=[
         'pynwb', 'tqdm', 'natsort', 'numpy', 'scipy', 'bs4',
-        'pandas', 'jupyter', 'matplotlib', 'h5py', 'pyyaml', 'jupyter-client',
+        'pandas', 'jupyter', 'matplotlib', 'h5py', 'pyyaml',
```

Comment:
@bendichter I changed the order here because I was getting some conflicts otherwise